### DISCUSSION TITLE
How do I view the Garbage Collection rules for the builder?

### DISCUSSION THREAD

**Discussion by musebc** [docker/buildx#1593] [Q&A] [answered]
I was looking through [the documentation](https://docs.docker.com/build/cache/garbage-collection)  for the builder Garbage Collector and I noticed a nice print out in the [Default Policies](https://docs.docker.com/build/cache/garbage-collection/#default-policies) section for the existing GC rules. I'm wondering if there is any way to print that out for a given server that has the docker engine and builder installed?

**Comment by crazy-max** [comment, marked as answer]
This should solve your issue: https://github.com/docker/buildx/pull/1440

**Reply by musebc** [reply]
When will that be merged?

**Reply by musebc** [reply]
Sorry. I just noticed that it was set for review last week. I'll keep an eye on it and check again once it's released. Thanks so much!
